app/uk/gov/hmrc/tai/model/domain/Person.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.tai.model.domain
+
+import org.joda.time.LocalDate
+import play.api.libs.json.{Format, Json}
+import uk.gov.hmrc.domain.Nino
+
+
+case class Person(nino: Nino,
+                  firstName: String,
+                  surname: String,
+                  dateOfBirth: Option[LocalDate],
+                  address: Address,
+                  isDeceased: Boolean,
+                  hasCorruptData: Boolean)
+
+object Person {
+  implicit val personFormat: Format[Person] = Json.format[Person]
+}
+
+case class Address(line1: String,
```

Comment:
Do we use address in the FE? If not we shouldn't have this in the model

Reply:
We can remove this during or after the second pr

Reply:
Yeah - will remove Address, and DOB fields on the latest PR.